### PR TITLE
Adding Case Study page and first case study

### PR DESCRIPTION
**Pull request by kocsmy**

### CODE REVIEW

**Comment by arikfr** [comment]
Can we switch places (in the logos) between HackerRank and Atlassian?

**Comment by kocsmy** [comment]
Sure!

**Comment by kocsmy** [comment]
Also added to the homepage.

**Comment by arikfr** [comment]
Is there a link from Customers page to the case study? Maybe we can put it next to Danya's blog post link?

File: website/customers.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Redash Customers
-description: Great collaction of our users
+title: Redash customers
```
Comment (arikfr) [inline website/customers.html]: `Redash *C*ustomers`

Although describing this as "Redash Users" is more accurate. Just not sure if it "sounds" right.

File: website/customers.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+description: Great collection of our customers
```
Comment (arikfr) [inline website/customers.html]: Let's change it to: "Empowering data-driven decision making in thousands of teams across the globe. From small teams to huge organizations, Redash helps tens of thousands of users to query and visualize data in a new and meaningful way."

Comment (kocsmy) [reply website/customers.html]: I stopped using title case by the way so I'd stick to Redash customers

**Pull request merged**